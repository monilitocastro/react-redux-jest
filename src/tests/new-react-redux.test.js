import React from 'react';
import {shallow} from 'enzyme';
import renderer from 'react-test-renderer';
import NewReactRedux from "../components/new-react-redux";
import {Provider} from 'react-redux';
import store from '../store';

describe('NewReactRedux', () => {
    let tree = null;
    beforeEach(() => {
        tree = renderer.create (<Provider store={store}>
            <NewReactRedux></NewReactRedux>
        </Provider>).toJSON();


    });
    it("can render initially", () => {
        expect(tree).toMatchSnapshot();
    })
});


// import { render, screen } from '@testing-library/react';
// import App from './App';

// test('renders learn react link', () => {
// render(<App />);
// const linkElement = screen.getByText(/learn react/i);
// expect(linkElement).toBeInTheDocument();
// });
