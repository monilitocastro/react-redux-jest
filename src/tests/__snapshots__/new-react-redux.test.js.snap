// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewReactRedux can render initially 1`] = `
Array [
  <h1>
     
    0
  </h1>,
  <div
    style={
      Object {
        "display": "flex",
      }
    }
  >
    <button
      id="newMinus"
      onClick={[Function]}
      style={
        Object {
          "fontSize": "1.5rem",
          "height": "40px",
          "width": "40px",
        }
      }
      type="button"
    >
      -
    </button>
    <button
      id="newAdd"
      onClick={[Function]}
      style={
        Object {
          "fontSize": "1.5rem",
          "height": "40px",
          "width": "40px",
        }
      }
      type="button"
    >
      +
    </button>
  </div>,
]
`;
